community/l10n_community_roles.md

```diff
@@ -0,0 +1,72 @@
+
+This document defines the roles within each l10n community, as well as the responsibilities and expectations of each. We also define what the l10n-driver responsibilities and expectations are with regard to the community as a whole. Here we define responsibilities as what each individual commits to by accepting their role in the community. We define expectations as how you carry yourself in the role.
+
+## L10n-drivers team (Mozilla staff)
+
+*Responsibilities*
+* 
+
+*Expectations*
+*
+
+## Locale manager
+
+*Responsibilities*
+
+* Establish mentorship practices for new localizers that foster growth and lead to impactful participation.
+* Identify and train community mentors in their role.
+* Coordinate translation and l10n testing of projects within the l10n community.
+* With input and consultation with Translators, evaluate candidates for increased roles within the community.
+* Coordinate and facilitate internal community discussions.
+* Create and maintain language resources for the community (e.g., terminology, style guides, etc.)
+
+*Expectations*
+
+* Open and frequent communication (liaison between l10n-drivers and l10n community) concerning new projects, new deadlines, and community issues.
+* Create and secure a safe space for people to participate in l10n in an impactful way.
+* Recruit and mentor new localizers.
+* Give timely feedback to community mentors and translators.
+
+## Translator
+
+*Responsibilities*
+
+* Submit translations for new source language content.
+* Review pending translation suggestions from contributors and provide them with timely feedback that focuses on developing skills.
+* In-context l10n testing of all submitted translations.
+* Work with managers to create and maintain language resources for the community.
+* File bugs concerning source language content issues.
+
+*Expectations*
+
+* Ensure consistent quality of the project they’re localizing.
+* Give detailed feedback to contributors in a timely manner.
+* Translate new content regularly.
+* Ensure that community style guidelines are being kept in translations that make it to the repositories.
+* Keep pending translation suggestion queue to a low number.
+* Gracefully accept feedback on their translations from other translators and managers.
+
+## Contributor
+
+*Responsibilities*
+
+* Submit translation suggestions to Pontoon projects for the target language.
+* Request and accept feedback from Translators and Managers in the community.
+* In-context review (l10n testing) of translation suggestions that have been approved by Translators and Managers.
+
+*Expectations*
+
+* Gracefully accept feedback on their translations from Translators and Managers.
+* Avoid using machine translation.
+* Proactively request feedback from Translators and Managers.
+* Doesn’t take shortcuts to increasing leaderboard standings.
```

Comment:
`Don’t` for consistency with other items? I'm always confused when it comes to "to + verb", my brain would expect "to increase".